LoopKit/CorrectionRangeOverrides.swift

```diff
@@ -113,8 +113,8 @@ extension CorrectionRangeOverrides: RawRepresentable {
     }
 }
 
-extension ClosedRange<HKQuantity> {
-    public func localizedDescription(unit: HKUnit) -> String {
-        String(format: NSLocalizedString("%.0f - %.0f %3$@", comment: ""), lowerBound.doubleValue(for: unit, withRounding: true), upperBound.doubleValue(for: unit, withRounding: true), unit.unitString)
+extension ClosedRange<LoopQuantity> {
```

Comment:
@ps2 here's a non naming change

Reply:
Why is the `withRounding` being removed? Is this used in UIs?

Reply:
Actually I think I can add this back now 👀

Reply:
Fixed